.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "python.pythonPath": "${workspaceFolder}/.venv/bin/python"
+  "python.pythonPath": "C:\\go-broke\\.venv\\Scripts\\python.exe",
```

Comment:
This is obviously not platform independent. Should we .gitignore this file?

Reply:
ya, good call